polar-core/src/error.rs

```diff
@@ -428,6 +431,17 @@ pub enum ValidationError {
         /// Term<Symbol> where the error arose, tracked for lexical context.
         term: Term,
     },
+    DuplicateShorthandRule {
+        resource: Term,
+        shorthand_rule: ShorthandRule,
+    },
```

Comment:
I thought we were punting on this for now?

Reply:
you are correct. I misinterpreted the "punt" in the quill thread. just backed the shorthand rule specific deduplication out.